app/views/home/index.html.erb

```diff
@@ -0,0 +1,64 @@
+<main>
+  <section class="spotlight">
+    <h2 class="spotlight__header">
+      <span class="spotlight__header--prefix">Media Spotlight:</span>
+
+      <a class="spotlight__link-to" href="/works/10"><%= link_to "#{Work.find_top_work.title}", work_path(Work.find_top_work.id) %></a> by <%= Work.find_top_work.creator %>
+    </h2>
+    <p class="spotlight__description"><%= Work.find_top_work.votes.count %> votes - <%= Work.find_top_work.description %>
+    </p>
+  </section>
+  <hr class="root__hr" />
+  <section class="top-ten__container">
+  <section class="top-ten__list-container">
+    <h2 class="top-ten__header">
+      Top Movies
+    </h2>
+    <ul class="list-group top-ten__list">
```

Comment:
You have the same code to show a list of works repeated 3 times. Could you use a view partial or a loop to DRY this up?